.github/scripts/tlAI-Linter.genai.js

```diff
@@ -0,0 +1,26 @@
+// metadata (including model parameters)
+// learn more at https://aka.ms/genaiscript
+script({ title: "tlAI-Linter", description: "Check if the prose comments and their TLA+ declarations and definitions are syntactically and semantically consistent" })
+
+// use def to emit LLM variables
+def("TLA+", env.files.filter(f => f.filename.endsWith(".tla")), {lineNumbers: true})
+
+// use $ to output formatted text to the prompt
+$`You are an expert at TLA+/TLAPLUS. Your task is to check if the prose comments and their TLA+ declarations and definitions are syntactically and semantically consistent!!!
```

Comment:
this is cool :D

Reply:
https://github.com/microsoft/tlaplus-examples/issues/3 and https://github.com/microsoft/tlaplus-examples/issues/1 show when and where it worked.